src/app/guides/how-to-get-funding-for-your-open-source-projects/how-to-get-funding-for-your-open-source-projects.mdx

```diff
@@ -0,0 +1,62 @@
+
+Direct sponsorship is the most straightforward way to get funding for your project.
+When a user or a different company wants to sponsor your project, they send the funds right to your project’s financial account.
+Here is a list of resources that provide direct sponsorships:
+
+- **GitHub Sponsors**: This is a popular option that creators use to fund their projects.
+  It allows you to choose the number of tiers you charge and rewards that you can offer to potential sponsors.
+  It also provides you with the opportunity to support other developers and organizations, which can be a good way to network with others and gain more support.
+  If you’re interested in using this resource, check out the official website for more information: [https://github.com/sponsors](https://github.com/sponsors)
+- **Open Collective**: This is a company that is devoted to supporting and promoting open source projects.
+  Like GitHub sponsors, they allow you to choose the number of tiers you charge and reward you with potential sponsors.
+  In contrast, Open Source Collective provides information on how the money is being used.
+  If interested, check out their website for more information: [https://opencollective.com/how-it-works](https://opencollective.com/how-it-works)
+
+## Aggregated sponsorships
+
+In contrast to direct sponsorships, aggregated sponsorships are usually provided by a company or organization that can pool funds together.
+The benefits of this option are that you can get financial support from a larger pool of people on a regular basis, which can be a good way to keep your project funded.
+Here is a list of resources that offer aggregated sponsorships:
+
+- **Tidelift**: In addition to providing tools to help you maintain your project, Tidelift offers maintainers various ways to receive financial support.
+  For example, you can connect your [GitHub Sponsors](https://tidelift.com/subscription/how-to-connect-tidelift-with-github) account to Tidelift and get paid based on the number of sponsors you have.
+  Want to learn more? Check out their post to see how they can help you with funding: [https://support.tidelift.com/hc/en-us/articles/4406294816916](https://support.tidelift.com/hc/en-us/articles/4406294816916)
+- **Thanks.dev**: This source offers funding distribution at different levels, offers you ownership on how your donation should be used and automates as well as notifies you about your donation to other projects being used.
+  Want to learn more? Check out their FAQ post here: **[https://thanks.dev/faq](https://thanks.dev/faq)**
+- **Stackaid**: In contrast to the above two sources, Stackaid is the first sponsor to use a subscription model to help owners gain funding for their open source projects.
+  If you’re interested in learning more about how it works, check out their website here: [https://www.stackaid.us/](https://www.stackaid.us/)
+
+## Legal paperwork needed for employees contributing to open source projects
```

Comment:
[Request] I think licensing is a wholly separate subject. And the phrasing of _"distribute to your employees"_ is confusing to me - what do you mean by that? I'd suggest just removing this section altogether, it's not really the topic this guide covers. (unless you feel strongly we should keep it in?)

Reply:
I guess licensing could be it's own section. I'll raise this as an issue.

Reply:
Coming from https://github.com/OpenContributionsProject/opencontributionssite/pull/141/files#r1233307403: I think what tripped me up is that this phrases the three potential licenses as examples of legal paperwork that, to me, reads as _"you could do some or all of these if you want"_. I'd suggest trimming this down to just mention the _types_ of paperwork one could have:

* A license file
* A sponsorship file
* A code of conduct
* ...other things that are common to repos

Thoughts?

Reply:
That's a good idea, and then each of those bullet points can be hyperlinked to explainers about these topics.